systemd/systemd.go

```diff
@@ -244,6 +244,10 @@ type Systemd interface {
 	Mask(service string) error
 	// Unmask the given service.
 	Unmask(service string) error
+	// Mount requests a mount of what under where with options.
+	Mount(what, where string, options ...string) error
+	// Umount requests a mount at where to be unmounted.
+	Umount(where string) error
```

Comment:
The systemd interface takes whatOrWhere in practice. This could lead to confusing results if we assume this is precisely where.

Reply:
Applied in #9565